### PR TITLE
feat(Skilavottord): Add mileage input

### PR DESCRIPTION
**Pull request by birkirkristmunds**
## What

Added mileage input field for recycling companies to enter mileage when car is brought to them. For now it is only set up in a temp files for testing purpose and to prevent breaking current functionality.

Can be tested on http://localhost:4200/app/skilavottord/deregister-vehicle-km

## Why

New law requires mileage to be entered when deregistering vehicles .

## Screenshots / Gifs

image
## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/skilavottord/web/screens/DeregisterVehicleKm/Confirm/ConfirmKM.tsx [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+`
+
+const Confirm: FC<React.PropsWithChildren<unknown>> = () => {
+  const {
+    control,
+    watch,
+    formState: { errors },
+  } = useForm({
+    mode: 'onChange',
+  })
```
Comment (veronikasif) [inline apps/skilavottord/web/screens/DeregisterVehicleKm/Confirm/ConfirmKM.tsx:89]: `formState: { errors },` is unused, can be removed
```suggestion
  const { control, watch } = useForm({
    mode: 'onChange',
  })
```

File: apps/skilavottord/web/screens/DeregisterVehicleKm/Overview/Overview.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+    return () => {
+      observer.disconnect()
+    }
+  }, [loading])
+
+  const handleDeregister = () => {
+    console.log(
+      'routes.deregisterVehicle.select',
+      routes.deregisterVehicle.select,
+    )
```
Comment (veronikasif) [inline apps/skilavottord/web/screens/DeregisterVehicleKm/Overview/Overview.tsx:127]: Remove console log
```suggestion
```

**Review by veronikasif — APPROVED** [review]

File: apps/skilavottord/web/screens/DeregisterVehicleKm/Confirm/ConfirmKM.tsx (hunk 1 above)
Comment (birkirkristmunds) [reply apps/skilavottord/web/screens/DeregisterVehicleKm/Confirm/ConfirmKM.tsx]: Fixed

File: apps/skilavottord/web/screens/DeregisterVehicleKm/Overview/Overview.tsx (hunk 2 above)
Comment (birkirkristmunds) [reply apps/skilavottord/web/screens/DeregisterVehicleKm/Overview/Overview.tsx]: Fixed

**Pull request merged**